### PR TITLE
new: EveriaClub.com (unoriginal)

### LINKED ISSUE #1765: EveriaClub (unoriginal, gravure)
### Source name

Everia Club

### Source link

https://www.everiaclub.com/

### Source language

All

### Other details

`Everiaclub.com` seems to be a clone of `everia.club`

It might be a good to have both of them, since `everia.club` regularly lost their old content due to using third party hosting. 

### Acknowledgements

- [X] I have checked that the extension does not already exist by searching the [extension listing](https://keiyoushi.github.io/extensions/) and verified it does not appear there.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a meaningful title with the source name.
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by cuong-tran**
This source host content their own, but only keep recent posts from original Everia
Close #1765

Checklist:

- [X] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [X] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [X] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [ ] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [X] Have tested the modifications by compiling and running the extension through Android Studio
- [X] Have removed `web_hi_res_512.png` when adding a new extension

### CODE REVIEW

File: src/all/everiaclubcom/src/eu/kanade/tachiyomi/extension/all/everiaclubcom/EveriaClubCom.kt [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+            }
+            status = SManga.COMPLETED
+        }
+    }
+
+    override fun fetchChapterList(manga: SManga): Observable<List<SChapter>> {
+        val chapter = SChapter.create().apply {
+            url = manga.url
+            name = "Gallery"
+            chapter_number = -2f
```
Comment (AwkwardPeak7) [inline src/all/everiaclubcom/src/eu/kanade/tachiyomi/extension/all/everiaclubcom/EveriaClubCom.kt]:
```suggestion
            chapter_number = 1f
```

File: src/all/everiaclubcom/src/eu/kanade/tachiyomi/extension/all/everiaclubcom/EveriaClubCom.kt [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    override val supportsLatest = true
+
+    override fun headersBuilder() = super.headersBuilder()
+        .add("Referer", "$baseUrl/")
+
+    private val Element.imgSrc: String
+        get() = attr("data-original")
+            .ifEmpty { attr("data-lazy-src") }
+            .ifEmpty { attr("data-src") }
+            .ifEmpty { attr("src") }
```
Comment (AwkwardPeak7) [inline src/all/everiaclubcom/src/eu/kanade/tachiyomi/extension/all/everiaclubcom/EveriaClubCom.kt]: wouldn't checking with `hasAttr` be better?

Comment (cuong-tran) [reply src/all/everiaclubcom/src/eu/kanade/tachiyomi/extension/all/everiaclubcom/EveriaClubCom.kt]: like this?
```
    private val Element.imgSrc: String
        get() = when {
            hasAttr("data-original") -> attr("data-original")
            hasAttr("data-lazy-src") -> attr("data-lazy-src")
            hasAttr("data-src") -> attr("data-src")
            hasAttr("src") -> attr("src")
            else -> ""
        }
```

Comment (AwkwardPeak7) [reply src/all/everiaclubcom/src/eu/kanade/tachiyomi/extension/all/everiaclubcom/EveriaClubCom.kt]: sure, and make nullable so else returns null

**Review by AwkwardPeak7 — APPROVED** [review]

**Pull request merged**